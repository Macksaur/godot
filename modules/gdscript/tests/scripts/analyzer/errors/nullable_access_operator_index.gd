class A:
    var b: A?
    var c: String?
    func banana():
        pass
func test():
    var foo := A.new()
    foo.c?[0].begins_with("")
