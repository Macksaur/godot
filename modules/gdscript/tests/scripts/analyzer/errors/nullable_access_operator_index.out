GDTEST_ANALYZER_ERROR
Potentially unsafe for nullable "<not implemented>" to call "begins_with()".
